.packit.yaml

```diff
@@ -0,0 +1,32 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+specfile_path: scap-security-guide.spec
+
+# add or remove files that should be synced
+synced_files:
+    - scap-security-guide.spec
+    - .packit.yaml
+
+# name in upstream package repository/registry (e.g. in PyPI)
+upstream_package_name: scap-security-guide
+# downstream (Fedora) RPM package name
+downstream_package_name: scap-security-guide
+
+actions:
+  post-upstream-clone: "wget https://src.fedoraproject.org/rpms/scap-security-guide/raw/rawhide/f/scap-security-guide.spec -O scap-security-guide.spec"
+
+
+# TODO: Remove the "jobs" key as soon as F34 is out
```

Comment:
My understanding was that this key disables F32 builds so I would say that it should be disabled after F32 is removed from the infrastructure. Am I correct or are you correct?

Reply:
Also, is F33 under fedora-development?

Reply:
I think that both of us are correct - when F34 is released, F32 will disappear, and we can return to default jobs.

F33 is clearly not a development version, as you can see when you check out test results. I have tried to avoid mentioning explicit version numbers, so things work even if nobody removes the `jobs` key.